### PR TITLE
Enable to use placeholder for schema_path

### PR DESCRIPTION
**Pull request by porkbeans**
Placeholder for schema_path #148 !

### CODE REVIEW

File: lib/fluent/plugin/out_bigquery.rb [hunk 1]
```diff
@@ -213,9 +213,6 @@ def configure(conf)
         if @schema
           @table_schema.load_schema(@schema)
         end
-        if @schema_path
-          @table_schema.load_schema(MultiJson.load(File.read(@schema_path)))
-        end
```
Comment (porkbeans) [inline lib/fluent/plugin/out_bigquery.rb]: No need to load schema at this time.

File: lib/fluent/plugin/out_bigquery.rb [hunk 2]
```diff
@@ -246,6 +243,7 @@ def start
         @tables_mutex = Mutex.new
         @fetched_schemas = {}
         @last_fetch_schema_time = Hash.new(0)
+        @read_schemas = {}
```
Comment (porkbeans) [inline lib/fluent/plugin/out_bigquery.rb]: Schemas loaded from schema_path.

File: lib/fluent/plugin/out_bigquery.rb [hunk 3]
```diff
@@ -355,9 +355,27 @@ def fetch_schema_target_table(metadata)
...
       end
 
+      def read_schema(metadata)
+        schema_path = read_schema_target_path(metadata)
+
+        unless @read_schemas[schema_path]
+          table_schema = Fluent::BigQuery::RecordSchema.new("record")
+          table_schema.load_schema(MultiJson.load(File.read(schema_path)))
+          @read_schemas[schema_path] = table_schema
+        end
```
Comment (porkbeans) [inline lib/fluent/plugin/out_bigquery.rb]: Placeholders for schema_path! yay!

**Comment by joker1007** [comment]
Thanks!!
It's looks good to me.

**Comment by joker1007** [comment]
Would you fix test codes?

**Pull request closed without merging**